.github/workflows/release.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - next
```

Comment:
I think calling the next branch `dev` would also make sense, but both work

Reply:
I want to strictly keep it as `next` major things go there, not things in development. So `dev` might be confusing, WDYT?

Reply:
Okay makes sense. I just immediately thought of next.js haha but this sounds reasonable